common/utils/filters.ts

```diff
@@ -0,0 +1,97 @@
+import {
+  CatalogueAggregationBucket,
+  CatalogueAggregations,
+} from '@weco/common/model/catalogue';
+import { nodeListValueToArray, inputValue } from '@weco/common/utils/forms';
+import { LinkProps } from 'next/link';
+
+import {
+  worksLink,
+  imagesLink,
+} from '@weco/common/services/catalogue/ts_routes';
+
+export const getFilterByCount = (
```

Comment:
I'm not sure I understand this name - isn't this `getNonEmptyFilters`?

Reply:
Makes sense with a better name of function :+1: